app/models/tv_season.rb

```diff
@@ -25,7 +26,8 @@ def self.parse_record(series:, show_id:, season_data:)
     season_id: season_data[:id],
     poster_path: season_data[:poster_path],
     season_number: season_data[:season_number],
-    credit: season_data[:credits],
+    credits: season_data[:credits],
+    cast_members: TVCastMember.parse_records(season_data[:credits][:cast]),
```

Comment:
We have access to the cast members for any given season! 😸 This will fix the Pulaski Glitch™️